### PR TITLE
feature: Configure eslint and prettier

### PR DESCRIPTION
**Pull request by i-am-monza**
This included adding a .eslint json file and updating rules as well. Added prettier package and .prettier file on src for formatting code. Additional rules where included in package.json file for eslint

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
```
Comment (hyperxtend) [inline package.json]: Add coverage to test script

Comment (hyperxtend) [reply package.json]: also, run lint script after the test to ensure there are no linting errors

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "airbnb",
+        "prettier",
+        "prettier/react"
+    ],
+    "globals": {
```
Comment (hyperxtend) [inline .eslintrc.json:14]: You could enable if the shallow, mount & render global variables for testing

File: package.json [hunk 3]
```diff
@@ -14,7 +14,8 @@
...
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint src/**/*.js"
   },
```
Comment (hyperxtend) [inline package.json:26]: Add a jest key after scripts to ignore certain files when running the test  
E.g: `"jest": {
		"coveragePathIgnorePatterns": [
			"/src/serviceWorker.js",
			"/src/index.jsx"
		]
	}`

**Review by hyperxtend — COMMENTED** [review]
Looks all good, enabling the globals will come into play once we start testing. So its no biggie right now

**Comment by i-am-monza** [comment]
> Looks all good, enabling the globals will come into play once we start testing. So its no biggie right now

I missed this notification. And here I have been waiting for this review - should have come here

**Pull request merged**